website/versioned_docs/version-0.13.0/flink-quick-start-guide.md

```diff
@@ -97,7 +97,7 @@ dependency to your project:
 <dependency>
     <groupId>org.apache.hudi</groupId>
     <artifactId>hudi-flink1.13-bundle</artifactId>
-    <version>0.13.0/version>
+    <version>0.13.0</version>
```

Comment:
Thanks, can you also fix the other versions?

Reply:
I checked the other versions and found no similar errors.